src/plugins/intel_cpu/tests/unit/shape_inference_test/scatter_update_shape_inference_test.cpp

```diff
@@ -0,0 +1,134 @@
+                                             StaticShape{1000, 125, 20, 10, 15},
+                                             StaticShape{1}},
+                             output_shapes = {StaticShape{}};
+    shape_inference(scatter_update.get(), input_shapes, output_shapes, constant_data);
+    EXPECT_EQ(output_shapes[0], StaticShape({1000, 256, 10, 15}));
+}
+
+TEST(StaticShapeInferenceTest, ScatterUpdate_axis_as_const) {
+    auto data_param = std::make_shared<ov::op::v0::Parameter>(element::i32, PartialShape{-1, -1, -1, -1});
+    auto indices_param = std::make_shared<ov::op::v0::Parameter>(element::i32, PartialShape{-1, -1});
+    auto updates_param = std::make_shared<ov::op::v0::Parameter>(element::i32, PartialShape{-1, -1, -1, -1, -1});
+    auto axis_const = std::make_shared<ov::op::v0::Constant>(element::i32, Shape{1}, std::vector<int32_t>{1});
+
+    auto scatter_update = std::make_shared<op::v3::ScatterUpdate>(data_param, indices_param, updates_param, axis_const);
+
+    std::vector<StaticShape> input_shapes = {StaticShape{1000, 256, 10, 15},
+                                             StaticShape{125, 20},
+                                             StaticShape{1000, 125, 20, 10, 15},
+                                             StaticShape{1}},
+                             output_shapes = {StaticShape{}};
+    shape_inference(scatter_update.get(), input_shapes, output_shapes);
+    EXPECT_EQ(output_shapes[0], StaticShape({1000, 256, 10, 15}));
+}
+
+TEST(StaticShapeInferenceTest, ScatterUpdate_dynamic_rank) {
+    auto data_param = std::make_shared<ov::op::v0::Parameter>(element::i32, PartialShape::dynamic());
+    auto indices_param = std::make_shared<ov::op::v0::Parameter>(element::i32, PartialShape::dynamic());
+    auto updates_param = std::make_shared<ov::op::v0::Parameter>(element::i32, PartialShape::dynamic());
+    auto axis_param = std::make_shared<ov::op::v0::Parameter>(element::i32, PartialShape::dynamic());
+
+    auto scatter_update = std::make_shared<op::v3::ScatterUpdate>(data_param, indices_param, updates_param, axis_param);
+
+    int32_t axis_val[] = {1};
+    std::map<size_t, std::shared_ptr<ngraph::runtime::HostTensor>> constant_data;
+    constant_data[3] = std::make_shared<ngraph::runtime::HostTensor>(ngraph::element::Type_t::i32, Shape{1}, axis_val);
+    std::vector<StaticShape> input_shapes = {StaticShape{1000, 256, 10, 15},
+                                             StaticShape{125, 20},
+                                             StaticShape{1000, 125, 20, 10, 15},
+                                             StaticShape{1}},
+                             output_shapes = {StaticShape{}};
+    shape_inference(scatter_update.get(), input_shapes, output_shapes, constant_data);
+    EXPECT_EQ(output_shapes[0], StaticShape({1000, 256, 10, 15}));
+}
+
+TEST(StaticShapeInferenceTest, ScatterUpdate_params_dynamic_rank_incorrect_updates_shape) {
```

Comment:
As these test focus on `shape_inference` don't see benefit to create operator from dynamic shape as it even not use same function for inference as used by `shape_inference`. The `shape_inference` is not taking even shapes from created operator (only number of outputs is required to be correct). 
Dynamic rank has no meaning when calling `shape_inference`.

It would be more beneficial to move this creation to `type_prop` as specific case of shape inference during op creation and check then all properties of created operator.

Ticket 90795 mention about default ctor testing. consider add test?

Reply:
The name of this test is `dynamic_rank_incorrect_updates_shape` and I've left the comments to emphasize that the shape of the third input (updates) is incorrect. 
Master branch version would fail during `shape_inference` step as it uses "fallback" to `validate_and_infer_types` of ScatterBase. 

So the goal of this test is to show that `shape_infer` implemented by `entryFirstPassthrough` doesn't validate the correctness of inputs rank and doesn't throw in this case.
Dynamic ranks for Parameters are used to avoid triggering shape rank validation at the op creation step, 
and test the `shape_inference` indeed.

-------------------------
About the default constructor testing, I thought that the purpose of such tests is to check correctness of setters, while ScatterUpdate has no attributes.